backend/src/main/kotlin/org/loculus/backend/controller/ExceptionHandler.kt

```diff
@@ -45,21 +43,19 @@ class ExceptionHandler : ResponseEntityExceptionHandler() {
         )
     }
 
-    @ExceptionHandler(DummyUnauthorizedExceptionToMakeItAppearInSwaggerUi::class)
```

Comment:
oh interesting - this function apparently is not actually used at runtime and will only show up in the swaggerUI